wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/devfile/validator/DevfileSchemaValidatorTest.java

```diff
@@ -50,7 +50,7 @@ public void shouldNotThrowExceptionOnValidationOfValidDevfile(String resourceFil
       {"editor_plugin_component/devfile_editor_component_with_custom_registry.yaml"},
       {"editor_plugin_component/devfile_editor_plugins_components_with_resource_limits.yaml"},
       {"editor_plugin_component/devfile_plugin_components_with_preferences.yaml"},
-      {"kubernetes_openshift_component/devfile_kubernetes_component_reference.yaml"},
```

Comment:
Is `devfile_kubernetes_component_reference.yaml` used anywhere?

Reply:
just file rename